ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py

```diff
@@ -74,7 +74,7 @@
     'powershell MSSQL Job',
     'powershell MSSQL AO AG',
     'powershell MSSQL AO AR',
-    'powershell MSSQL AO AL',
+    'powershell AO AL',
```

Comment:
As [here](https://github.com/zenoss/ZenPacks.zenoss.Microsoft.Windows/blob/cff575638d3d6b0f65d059304768f12a92753007/ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py#L1544) all strategies which starts with 'powershell MSSQL' use SQL Connection, and Availability Listener strategy doesn't require SQL Connection, so  I renamed it accordingly.